assets/navigations/components/EditNavigation.jsx

```diff
@@ -19,7 +19,7 @@ class EditNavigation extends React.Component {
         this.state = {activeTab: 'navigation-details'};
 
         this.tabs = [
-            {label: gettext('Navigation'), name: 'navigation-details'},
+            {label: gettext('Global Topic'), name: 'navigation-details'},
```

Comment:
imo this one should be `Global Topics`

Reply:
![image](https://user-images.githubusercontent.com/62991760/172111145-7733709b-85d0-4193-833c-df1ff6e6a8d0.png)
Here, since we are editing single global topic, I kept it singular.